packages/venia-concept/src/components/Checkout/form.js

```diff
@@ -222,26 +202,56 @@ class Form extends Component {
         );
     }
 
+    get shippingAddressSummary() {
+        const { classes, haveShippingAddress, shippingAddress } = this.props;
+
+        if (!haveShippingAddress) {
+            return (
+                <span className={classes.informationPrompt}>
+                    Add Shipping Information
+                </span>
+            );
+        }
+
+        const name = `${shippingAddress.firstname} ${shippingAddress.lastname}`;
+        const street = `${shippingAddress.street.join(' ')}`;
+
+        return (
+            <Fragment>
+                <strong>{name}</strong>
+                <br />
+                <span>{street}</span>
+            </Fragment>
+        );
+    }
+
     get shippingMethodSummary() {
-        const {
-            classes,
-            isPaymentMethodReady,
-            isShippingMethodReady,
-            shippingTitle
-        } = this.props;
+        const { classes, haveShippingMethod, shippingTitle } = this.props;
 
-        if (!isShippingMethodReady) {
-            const promptClass = isPaymentMethodReady
-                ? classes.informationPrompt
-                : classes['informationPrompt--disabled'];
+        if (!haveShippingMethod) {
             return (
-                <span className={promptClass}>Add Shipping Information</span>
+                <span className={classes.informationPrompt}>
+                    Add Shipping Information
+                </span>
             );
         }
 
+        const twoDaysInMilliseconds = 1000 * 60 * 60 * 24 * 2;
+        const twoDaysFromNowDate = new Date(Date.now() + twoDaysInMilliseconds);
+        const twoDaysFromNowDisplay = twoDaysFromNowDate.toLocaleDateString(
+            'en-US',
+            {
+                day: 'numeric',
+                month: 'long',
+                year: 'numeric'
+            }
+        );
+
         return (
             <Fragment>
-                <strong>{shippingTitle}</strong>
+                <strong>{twoDaysFromNowDisplay}</strong>
+                <br />
+                <span>{shippingTitle}</span>
```

Comment:
I've already discussed with @soumya-ashok why we shouldn't do this.

If the shopper is ordering on a Friday, is this telling them it'll be there by Sunday? Should this function be aware of holidays? Does it know if the shipper ships on weekends? Of course not.

"Estimated ship date" is not a mathematical calculation; it's a logistical one. Worse, a shopper is likely to interpret it as a guarantee, even if we were to add an explicit disclaimer that it's not—and they'll be upset accordingly.

If we get a ship date back from the API, that's one thing, but this shouldn't be client-side. 😅

Reply:
Good points, I was unaware!

Reply:
@supernova-at I'm sorry I missed the preset ship date - I was too caught up with all the other details. Thanks for your work on this!

Reply:
I will update this to match the mock:

<img width="376" alt="screen shot 2019-02-01 at 11 06 56 am" src="https://user-images.githubusercontent.com/13182778/52137979-85a51980-2611-11e9-8a9d-b9bf45d5405d.png">

Reply:
Resolved in https://github.com/magento-research/pwa-studio/pull/799/commits/7244054a1cbbf369e19126a3e22b77fa0800d24c.